tfa.services.yml

```diff
@@ -2,6 +2,9 @@ services:
   plugin.manager.tfa.login:
     class: Drupal\tfa\TfaLoginPluginManager
     parent: default_plugin_manager
+  plugin.manager.tfa.base:
```

Comment:
Hmm... I don't think 'base' is a plugin manager, is it? Should this be here?

I think this should be removed